modules/cluster-logging-log-forwarding-configure.adoc

```diff
@@ -0,0 +1,242 @@
+* Specify insecure: true if the endpoint does not use a secret, resulting in insecure communication.
+<3> Add one or more pipelines:
+* Enter a name for the pipeline
+* Specify the source type, logs.app, logs.infra, or logs.audit.
+* Specify the name of one or outputs configured in the CR.
+
+[NOTE]
+====
+If you set `disableDefaultForwarding: true` you must configure a pipeline and output for all three types of logs, application, infrastructure, and audit. If you do not specify a pipeline and output for a log type, those logs are not stored and will be lost.
+==== 
+
+== Example log forwarding custom resources
+ 
+A typical Log Forwarding configuration would be similar to the following examples.
+
+The following Log Forwarding custom resource sends all logs to a secured external Elasticsearch instance:
+
+.Sample custom resource to forward to an Elasticsearch instance
+
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogForwarding
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  disableDefaultForwarding: true
+  outputs:
+    - name: user-created-es
+      type: elasticsearch
+      endpoint: 'elasticsearch-server.openshift-logging.svc:9200'
+      secret:
+        name: piplinesecret
+  pipelines:
+    - name: app-pipeline
+      inputSource: logs.app
+      outputRefs:
+        - user-created-es
+    - name: infra-pipeline
+      inputSource: logs.infra
+      outputRefs:
+        - user-created-es
+    - name: audit-pipeline
+      inputSource: logs.audit
+      outputRefs:
+        - user-created-es
+----
+
+The following Log Forwarding custom resource sends all logs to a secured Fluentd instance using the Fluentd `out_forward` plug-in.
+
+.Sample custom resource to use the`out_forward` plugin
+
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogForwarding
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  disableDefaultForwarding: true
+  outputs:
+    - name: fluentd-created-by-user
+      type: forward
+      endpoint: 'fluentdserver.openshift-logging.svc:24224'
+      secret:
+        name: fluentdserver
+  pipelines:
+    - name: app-pipeline
+      inputType: logs.app
+      outputRefs:
+        - fluentd-created-by-user
+    - name: infra-pipeline
+      inputType: logs.infra
+      outputRefs:
+        - fluentd-created-by-user
+    - name: clo-default-audit-pipeline
+      inputType: logs.audit
+      outputRefs:
+        - fluentd-created-by-user
+----
+
+The following Log Forwarding custom resource sends all logs to the cluster-logging-managed Elaticsearch instance, the default log forwarding method.
+
+.Sample custom resource to use the default log forwarding
+
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1alpha1
+kind: LogForwarding
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  disableDefaultForwarding: false
+----
```

Comment:
I think no need to write this example here. Creating a LF instance using this yaml file will be failed since outputs and pipelines are required https://github.com/openshift/cluster-logging-operator/blob/release-4.3/manifests/4.3/logforwardings.crd.yaml#L75-L77.
Please remove https://github.com/openshift/openshift-docs/pull/17825/files#diff-425dab1c9ddf18e7941c3707f617566bR167-R180